oneview_redfish_toolkit/mockups/redfish/ResourceBlockEthernetInterface.json

```diff
@@ -1,6 +1,6 @@
 {
   "@odata.type": "#EthernetInterface.v1_3_0.EthernetInterface",
-  "Id": "1f0ca9ef-7f81-45e3-9d64-341b46cf87e0",
+  "Id": "1",
```

Comment:
Should we have unit tests here, not? :thinking: Evaluate the possibility.

Reply:
The test is already covering the scenario, for this reason I have changed the mockup json.

Reply:
Perfect!